Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM ubuntu:bionic
+
+# Set up dependencies
+ENV GCC_VER "10"
+ENV LLVM_VER "11"
+ENV CC gcc-${GCC_VER}
+ENV CXX g++-${GCC_VER}
```

Comment:
Perhaps `CC` and `CXX` should be `ARG` and not `ENV`, so they can be passed as arguments to `docker build`?